docs/devguide/design.rst

```diff
@@ -1,9 +1,4 @@
-Design and Rationale
-
-
-
-Swift vs Parsl
---------------
+Historical: Swift vs Parsl
```

Comment:
Looks like we forgot to keep the heading moniker.

Reply:
added